constants/constants.F90

```diff
@@ -37,12 +37,24 @@ module constants_mod
 
 real :: realnumber !< dummy variable to use in HUGE initializations
 
+#ifdef SMALL_EARTH
+#if defined(DCMIP) || (defined(HIWPP) && defined(SUPER_K))
+ real, public, parameter :: small_fac =  1._r8_kind / 120._r8_kind #only needed for supercell test
```

Comment:
Should the comment on this have a ! instead of a #

Reply:
Yes it should, although an error will only be triggered in the presence of the proper macros.

Reply:
It's a good thing I had on my glasses then.